multidimensional_sampling/README.md

```diff
@@ -0,0 +1,233 @@
+# Multidimensional Sampling
+
+This repo contains a pgen script for maestro that implements several
+multidimensional sampling methods which are driven by a simple yaml
+specification included in the env/variables section of any maestro
+specification. This yaml specification is a nested list named
+`SAMPLE_DICTIONARY`, and this repo contains several examples. 
+
+The pgen script is executed as follows
+ 
+```
+maestro run {-y} --pgen ./pgen_sample.py {sample_list.yaml},
+```
+
+where {-y} is an optional `auto-run` option, and `sample_list.yaml` is a maestro specification containing a `SAMPLE_DICTIONARY`.  
+
+`pgen_sample.py` currently supports four sampling modes:
+
+1. `list`: see `sample_list.yaml`
+1. `column_list`: see `sample_column_list.yaml`
+1. `cross_product`: see `sample_cross_product.yaml`
+1. `best_candidate`: see `sample_best_candidate.yaml`
+
+Note: the `best_candidate` sampling mode currently requires that maestro is installed from scratch in a virtual environment (see https://github.com/LLNL/maestrowf#setting-up-your-python-environment) and that the `pandas`, `scipy`, and `sklearn` packages be installed in that environment as well. Please see the `Installing Maestro, pandas, scipy, and sklearn in a virtual environment` section below. 
```

Comment:
I'd revise the link to use markdown to use the link syntax -- something like: `(see Maestro [install documentation](https://github.com/LLNL/maestrowf#setting-up-your-python-environment)`

Reply:
Thanks! Please let me know if the following still needs fixing.

Note: the `best_candidate` sampling mode currently requires that maestro is installed from scratch in a virtual environment (see Maestro [install documentation](https://github.com/LLNL/maestrowf#setting-up-your-python-environment)) and that the `pandas`, `scipy`, and `sklearn` packages be installed in that environment as well. Please see the `Installing Maestro, pandas, scipy, and sklearn in a virtual environment` section below.

Reply:
This looks good -- I think there's a way to self-reference a section the same file. That would make it better for the `Installing Maestro, pandas,...` in-line code block style. This stackoverflow comment says you can self reference within a README: https://stackoverflow.com/a/38507669